dependencies/O2Dependencies.cmake

```diff
@@ -48,6 +48,13 @@ set_package_properties(ROOT PROPERTIES TYPE REQUIRED)
 find_package(fmt)
 set_package_properties(fmt PROPERTIES TYPE REQUIRED)
 
+find_package(PkgConfig)
+set_package_properties(PkgConfig PROPERTIES TYPE REQUIRED)
+
+# TODO: Pass this path as an -D cmake argument, avoiding `ubuntu2004_x86-64`
+set(ENV{PKG_CONFIG_PATH} "$ENV{ALIBUILD_WORK_DIR}/ubuntu2004_x86-64/ZeroMQ/latest/lib/pkgconfig")
```

Comment:
This clearly needs to go.

Reply:
I agree but I honestly don't know how to do it (I put some time into it but failed). That's why I put this comment.

Reply:
Ok, but this needs to be cleaned in any case. Why not simply using FairMQ, BTW?